### DISCUSSION TITLE
Frigate +

### DISCUSSION THREAD

**Discussion by dz4news** [blakeblackshear/frigate#4534] [Ask A Question] [answered]
When I used the HA Add On for Frigate, I was able to input an API key to activate an option to upload to Frigate+ from the application. Now that I am using Frigate in a docker container, this this process still possible or was it exclusive to the Add on? If it is possible, any examples of the code needed?

**Comment by NickM-27** [comment, marked as answer]
Just set it as an environment variable in the docker compose file or docker run 

`PLUS_API_KEY: xyz123`

**Reply by blakeblackshear** [reply]
It's actually `PLUS_API_KEY`
